docker/sentinel-compose.yml

```
@@ -0,0 +1,91 @@
+      ALLOW_EMPTY_PASSWORD: "yes"
+
+  redis-server-1:
+    image: falkordb/falkordb:latest
+    hostname: redis-server-1
+    container_name: redis-server-1
+    ports:
+      - 6380:6380
+    networks:
+      - redis-sentinel
+    environment:
+      REDIS_ARGS:  "--port 6380 --replica-announce-ip redis-server-1 --replica-announce-port 6380"
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)